src/js/login/containers/Main.jsx

```diff
@@ -75,6 +80,7 @@ class Main extends React.Component {
       });
       const receiver = window.open(`${myLoginUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
       receiver.focus();
+      this.getLoginUrl();
```

Comment:
Seems to me like it's creating two new sessions every time just in case the second one is going to be used in the future. Is that going to cause any trouble anywhere?